projects/systelab-components/src/lib/directives/keyup-debounce.directive.ts

```diff
@@ -0,0 +1,36 @@
+import { AfterContentInit, AfterViewInit, Directive, ElementRef, EventEmitter, HostListener, Input, OnDestroy, OnInit, Output } from '@angular/core';
```

Comment:
As a suggestion, I think it should be possible to create a unit test for this directive. 
Given a large enough debounce time, you could create a test harness with a control which used the directive, simulate a keyup event on the control (I think using dispatchEvent should work, though sometimes it doesn't), and verify that the output has not been called, then wait the debounce time and verify it has been called.

Reply:
done

Reply:
Great work with the unit test, thanks! 👏